tests/test_configuration/test_aws_ssm.py

```diff
@@ -0,0 +1,14 @@
+from unittest.mock import MagicMock, patch
+
+from lbz.aws_boto3 import client
+from lbz.configuration import SSM
+
+
+@patch.object(client.ssm, "get_parameter")
+def test_get_parameter(mocked_get_parameter: MagicMock) -> None:
```

Comment:
[#961442723](https://github.com/pdyba/lambdalizator/pull/63#discussion_r961442723):
```suggestion
def test__get_parameter__returns_value_fetched_from_aws(mocked_get_parameter: MagicMock) -> None:
```